### ISSUE TITLE
Get lts and quotes 

### ISSUE THREAD

**Issue by bibinvargheset** [Shoonya-Dev/ShoonyaApi-py#61]
if there was a function to get ltps and quotes of a list of instruments it would be very helpful for strategy implementations

def ltp(	self, *instruments)
Retrieve last price for list of instruments.

instruments is a list of instruments, Instrument are in the format of exchange:tradingsymbol. For example NSE:INFY

https://kite.trade/docs/pykiteconnect/v3/#kiteconnect.KiteConnect.ltp

**Comment by williamzerner** [comment]
You can get ltp by streaming the data using test_realtime.py and store it in csv which is located in the test folder. And then use another program to read the data stored with the help of pandas. To get in the format NSE:INFY... i have no clue but again you can use pandas to create your own format

**Comment by bibinvargheset** [comment]
> You can get ltp by streaming the data using test_realtime.py and store it in csv which is located in the test folder. And then use another program to read the data stored with the help of pandas. To get in the format NSE:INFY... i have no clue but again you can use pandas to create your own format

What are you replying to my issue was how to fetch multiple signal quotes /ltp in one call, I am not asking about realtime streaming please read the issue before repling

**Comment by williamzerner** [comment]
> What are you replying to my issue was how to fetch multiple signal quotes /ltp in one call, I am not asking about realtime streaming please read the issue before repling

you can use get_time_series function and then get the close of the latest candle. That will be your ltp. If you are asking about an inbuilt function , it doesnt exist.

**Comment by bibinvargheset** [comment]
Yes, thats why we are asking to create one, which will be very helpful for many API implementation and will reduce time delay and server calls as it is implemented in most other leading broker apis, like zerodha, upstocks , samco etc,

**Comment by DeepBhuptani** [comment]
```
instruments = ['NSE:INFY','NSE:INFY','NSE:INFY']
for n in range(len(instruments)):
    symbl = api.searchscrip(exchange = instruments[n][:3], searchtext = instruments[n][4:])
    tokn = symbl['values'][0]['token']
    print(api.get_quotes(exchange = instruments[n][:3], token = tokn)['lp'])
```

**Comment by bibinvargheset** [comment]
> ```
> instruments = ['NSE:INFY','NSE:INFY','NSE:INFY']
> for n in range(len(instruments)):
>     symbl = api.searchscrip(exchange = instruments[n][:3], searchtext = instruments[n][4:])
>     tokn = symbl['values'][0]['token']
>     print(api.get_quotes(exchange = instruments[n][:3], token = tokn)['lp'])
> ```

are you trying to demonstrate what python has for loops or how inefficient the current solution is by using the for loop it will take forever to get multiple quotes.

**Comment by CosmicTrader** [comment]
Agree  with bibinvargheset, this feature is very helpful when you are trading options specially highly volatiles ones like index, fetching prices for more individually is costly both in terms of time and server calls.
Team SHOONYA please make this feature available. And there bugs in many other APIs, please try to solve them fast.

**Comment by Anil-Dhruw** [comment]
**searchscrip()** function is not working 

res = api.searchscrip('NSE', 'RELI')
print(res)

output : 
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.shoonya.com:443
DEBUG:urllib3.connectionpool:[https://api.shoonya.com:443](https://api.shoonya.com/) "POST /NorenWClientTP/SearchScrip HTTP/1.1" 404 58
**DEBUG:NorenRestApiPy.NorenApi:{"stat":"Not_Ok","emsg":"No Data :  Exchange Not enabled"}**
None
